com.simutrans.Simutrans.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.simutrans.Simutrans
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: simutrans
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+modules:
+  - shared-modules/linux-audio/fluidsynth2.json
+  - name: bzip2
+    buildsystem: simple
+    build-commands:
+      - make install PREFIX=$FLATPAK_DEST
+    cleanup:
+      - /bin
+      - /include
+      - /lib
+    sources:
+      - type: archive
+        url: https://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz
+        sha256: ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0c4a2269
+
+# Issues were found downloading miniupnpc at the time of the 124.1 release. Maybe for the next one.
+#  - name: miniupnpc
+#    buildsystem: cmake-ninja
+#    config-opts:
+#        - -DUPNPC_BUILD_STATIC=FALSE
+#        - -DUPNPC_BUILD_SHARED=TRUE
+#        - -DUPNPC_BUILD_TESTS=FALSE
+#        - -DUPNPC_BUILD_SAMPLE=FALSE
+#    cleanup:
+#        - /include
+#        - /share/doc
+#    sources:
+#      - type: archive
+#        url: http://miniupnp.free.fr/files/miniupnpc-2.2.7.tar.gz
+#        sha256: 741175f3a6edb68aa2a5a62a576ffc5596f28d775e6be0a447a41d5881f145ee
+
+  - name: simutrans
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSIMUTRANS_USE_REVISION=11266
+    builddir: true
+    post-install:
+      - mv $FLATPAK_DEST/share/icons/hicolor/scalable/apps/simutrans.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/com.simutrans.Simutrans.svg
```

Comment:
you should use `rename-icon` instead. This would also make the desktop-file modification uncessary (Line 53)